drivers/pwm/pwm_numaker.c

```diff
@@ -432,6 +432,29 @@ static int pwm_numaker_clk_get_rate(EPWM_T *epwm, uint32_t *rate)
 	uint32_t clk_src;
 	uint32_t epwm_clk_src;
 
+#if defined(CONFIG_SOC_SERIES_M55M1X)
+	if (epwm == EPWM0) {
+		clk_src = CLK->EPWMSEL & CLK_EPWMSEL_EPWM0SEL_Msk;
+	} else if (epwm == EPWM1) {
+		clk_src = CLK->EPWMSEL & CLK_EPWMSEL_EPWM1SEL_Msk;
+	} else {
+		LOG_ERR("Invalid EPWM node");
+		return -EINVAL;
+	}
+
+	if (clk_src == 1U) {
```

Comment:
add a define for that `1U` mentioning `HCLK0` in the name

Reply:
Done & thanks.